### DISCUSSION TITLE
posts/low-power-efficient-server/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#286] [Comments]
# posts/low-power-efficient-server/

What if I told you that this little machine is the perfect Proxmox Virtualization server? And what if I told you I crammed an intel core i5, 64 GB of RAM, a 1 TB NVMe SSD, another 1TB SSD all in this tiny little box that’s dead silent without any fans? And what if I told you it can run Proxmox Virtual Server, host a pfsense router, runTrueNAS with a TB of storage, run Ubuntu server with Portainer running a few docker containers, run Windows 10 or Windows 11, and run ubuntu Desktop and pass though all of the hardware so I can use this server as a desktop, all while running a Plex Server and doing hardware transcoding with 3 - 4k streams?

https://technotim.live/posts/low-power-efficient-server/

**Comment by Ceredril** [comment]
I am surprised to see that no one has commented on this yet!

To make this short:
Could you point me into the right direction as to how you passed through the GPU to your VMs? Did you figure it out on your own? Are there guides you can recommend?

PS: Love your videos, might have bought one of those Rectli machines if they had 12th gen processors.
